amr-wind/physics/SyntheticTurbulence.cpp

```diff
@@ -449,15 +448,16 @@ SyntheticTurbulence::SyntheticTurbulence(const CFDSim& sim)
     //
     // x-direction points to flow direction (convert from compass direction to
     // vector)
-    m_turb_grid.tr_mat.xx() = -std::sin(wind_direction);
-    m_turb_grid.tr_mat.xy() = -std::cos(wind_direction);
+    m_turb_grid.tr_mat.xx() = std::cos(theta_t);
```

Comment:
What's the bug here?

Reply:
For some reason, the second row of `tr_mat` came out to zero. This seemed to fix that problem.

Reply:
Seems like there bug is in there cross product and not here.

Reply:
Also you can simplify this section by writing 

```C++
const amrex::Real wdir = 270.0 - wind_direction; 
m_turb_grid.tr_mat = Tensor::zrot(wdir);
```

Reply:
Thanks Shreyas. Fixed in https://github.com/Exawind/amr-wind/pull/399/commits/57de3963a2bf9632042441ceed12f74cde912cf5